bouncycastle/src/main/scala/tsec/Bouncy.scala

```diff
@@ -0,0 +1,16 @@
+package tsec
+
+import java.security.Security
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+
+trait Bouncy
+
+object Bouncy {
+  // @alexknvl
+  implicit val GetSchwifty: Bouncy = {
+    if (Security.getProvider("BC") == null)
+      Security.addProvider(new BouncyCastleProvider())
+    new Bouncy {}
+  }
```

Comment:
This is simultaneously disgusting and beautiful.

Reply:
I agree, but I really don't see a better way around this :(.

It's between that, and having people's code fail at runtime like [we do right now on the docs](https://jmcardon.github.io/tsec/docs/signatures.html).

I'm personally pro-fixing this yolostyle approach.

Reply:
Yes I'm :+1: on it. Alternatively make the init happen as a side effect of the module ctor of everything that needs it, but that's fragile.